### ISSUE TITLE
Deploy (experimental) apex class with a duplicate name as one in a managed package 

### ISSUE THREAD

**Issue by mnunezdm** [forcedotcom/salesforcedx-vscode#2188]
### Summary

I'm trying to deploy an Apex Class with a name `TestClass.cls` into an org which has a managed package installed that also has a `TestClass.cls`

### Steps To Reproduce:

1. Activate experimental deploy
1. Install a managed package
1. Try to deploy an Apex Class which has a duplicate name with one in that managed package

### Expected result

Deployed correctly

### Actual result

In case the class is not created

```
11:35:15.287 Starting Deploy (Beta)
=== Deploy Errors
11:35:17.766 Finished Deploy (Beta)
```

In case the class is already created

```
11:52:42.619 Starting Deploy (Beta)
When retrieving results with Metadata or FullName fields, the query qualificatio​ns must specify no more than one row for retrieval. Result size: 2
```

### Additional information

_Feel free to attach a screenshot_.

**VS Code Version**: v1.45

**SFDX CLI Version**:

```
sfdx plugins --core                                                                                                                                                                                       
@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.2.3 (core)
@oclif/plugin-help 2.2.3 (core)
@oclif/plugin-not-found 1.2.3 (core)
@oclif/plugin-plugins 1.7.9 (core)
@oclif/plugin-update 1.3.9 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.0.7 (core)
analytics 1.7.1 (core)
generator 1.1.2 (core)
salesforcedx 48.12.1 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.5 (core)
├─ salesforcedx-templates 48.10.0 (core)
└─ salesforce-alm 48.12.0 (core)

sfdx-cli 7.56.1 (core)
```

**OS and version**: Ubuntu 18.04.4 LTS in WSL 2

**Comment by benjaminloerincz** [comment]
Also, on retrieving the class the class of the managed package will be retrieved instead of the class of the non-managed package.

sfdx-cli/7.60.0-e2bb2907cb darwin-x64 node-v10.15.3

**Comment by lcampos** [comment]
The fix for both deploy and retrieve (experimental) has been released as part of 48.24.0. Thanks for providing feedback.
